packages/code/src/example1.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
It would be nice if you could turn these `example`s to full-on `tests`, so they can be in the repo and have a real place and purpose rather than just laying in the source folder. See `packages/html/tests` for examples.